proto/server/configEvtHandler.go

```diff
@@ -210,6 +215,34 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	if errPost != nil {
 		logger.DbLog.Warnln(errPost)
 	}
+	if factory.WebUIConfig.Configuration.SendPebbleNotifications {
+		err := sendPebbleNotification("aetherproject.org/webconsole/networkslice/create")
+		if err != nil {
+			logger.ConfigLog.Warnln("sending Pebble notification failed: %s. continuing silently.", err.Error())
+		}
+	}
+	rwLock.Unlock()
+}
+
+func handleNetworkSliceDelete(configMsg *configmodels.ConfigMessage, subsUpdateChan chan *Update5GSubscriberMsg) {
+	rwLock.Lock()
+	if factory.WebUIConfig.Configuration.Mode5G {
+		var config5gMsg Update5GSubscriberMsg
+		config5gMsg.Msg = configMsg
+		config5gMsg.PrevSlice = getSliceByName(configMsg.SliceName)
+		subsUpdateChan <- &config5gMsg
+	}
+	filter := bson.M{"SliceName": configMsg.SliceName}
+	errDelOne := dbadapter.CommonDBClient.RestfulAPIDeleteOne(sliceDataColl, filter)
+	if errDelOne != nil {
+		logger.DbLog.Warnln(errDelOne)
```

Comment:
With the refactor, no need for a different variable name for err:

```suggestion
	err := dbadapter.CommonDBClient.RestfulAPIDeleteOne(sliceDataColl, filter)
	if err != nil {
		logger.DbLog.Warnln(err)
```